mws/apis/outbound_shipments.py

```diff
@@ -32,15 +32,50 @@ def get_fulfillment_preview(self):
         """
         raise NotImplementedError
 
-    def create_fulfillment_order(self):
+    def create_fulfillment_order(self, marketplace_id = None, seller_fulfillment_order_id = None, fulfillment_action = None,
+                            displayable_order_id = None, displayable_order_datetime = None, displayable_order_comment = None,
+                            shipping_speed_category = None, destination_address = None, fulfillment_policy = None,
+                            notification_email_list = None, cod_settings = None, items = None, delivery_window = None):
         """
         Requests that Amazon ship items from the seller's inventory in Amazon's fulfillment network
         to a destination address.
 
+        :param marketplace_id:
+        :param seller_fulfillment_order_id: Required
+        :param fulfillment_action:
+        :param displayable_order_id: Required
+        :param displayable_order_datetime: Required
+        :param displayable_order_comment: Required
+        :param shipping_speed_category: Required
+        :param destination_address: Required
+        :param fulfillment_policy:
+        :param notification_email_list:
+        :param cod_settings:
+        :param items: Required
+        :param delivery_window:
+
         Docs:
         http://docs.developer.amazonservices.com/en_US/fba_outbound/FBAOutbound_CreateFulfillmentOrder.html
         """
-        raise NotImplementedError
+        data = {
+            "Action": "CreateFulfillmentOrder",
+            "MarketplaceId": marketplace_id,
+            "SellerFulfillmentOrderId": seller_fulfillment_order_id,
+            "FulfillmentAction": fulfillment_action,
+            "DisplayableOrderId": displayable_order_id,
+            "DisplayableOrderDateTime": displayable_order_datetime,
+            "DisplayableOrderComment": displayable_order_comment,
+            "ShippingSpeedCategory": shipping_speed_category,
+            "FulfillmentPolicy": fulfillment_policy,
+        }
+
+        data.update(utils.enumerate_keyed_param("Items.Item", items or []))
```

Comment:
Referring to [MWS documentation](http://docs.developer.amazonservices.com/en_US/fba_outbound/FBAOutbound_CreateFulfillmentOrder.html), this key is expected to be `Items.member.X.key`. Should change the keyed param argument to `Items.member`.